lua/neogit.lua

```diff
@@ -26,8 +26,18 @@ local neogit = {
     opts = opts or {}
 
     if not cli.git_is_repository_sync() then
-      notification.create("The current working directory is not a git repository", vim.log.levels.ERROR)
-      return
+      local create = vim.fn.input({
```

Comment:
This could use `vim.fn.confirm()` - there are some other examples of that in the codebase. Or, better yet, `require("neogit.lib.input").get_confirmation(...)`